app/controllers/admin/volunteers.py

```diff
@@ -42,8 +42,13 @@ def trackVolunteersPage(eventID):
 
     eventRsvpData = list(EventRsvp.select().where(EventRsvp.event==event))
     eventParticipantData = list(EventParticipant.select().where(EventParticipant.event==event))
-    eventVolunteerData = (eventParticipantData + eventRsvpData)
-
+    participantsAndRsvp = (eventParticipantData + eventRsvpData)
+    eventVolunteerData = []
+    volunteerUser = []
```

Comment:
What is the purpose of having these two new lists, it seems like the same info is being passed into each list. The only difference is one list is being passed the object and the other is being passed object.user

Reply:
The object list includes both EventRsvp and EventParticipant objects. So, they can't be compared. The user list appends the user, which is in both objects. That's how we were able to compare them.